tests/e2e/scale/noobaa/test_scale_endpoint.py

```diff
@@ -84,7 +84,8 @@ def test_scale_endpoint_and_respin_ceph_pods(
         self._assert_endpoint_count(desired_count=1)
 
         # Create s3 workload using mcg_job_factory
-        job = mcg_job_factory(custom_options=options)
+        for i in range(10):
```

Comment:
I tried similar code during noobaa db upgrade tests, please make sure test it once.
When I did this way buckets were created but object creations were slow might be due to lot of obj write to multiple buckets at same time.
Also ended up spending too much time for completion of obj write.
Would request to test this and if required sleep might work to avoid the problem

Reply:
I have tested this locally and it works fine. For Noobaa autoscale testing, the main thing is generating a high number of workload to ensure that nb endpoint CPU usage goes up to >80% for a period of time to trigger scale up (2 pods), then fio can be terminated to reduce the load so nb endpoint will be scaled down (back to 1 pod).  We don't need to wait for all FIO jobs to complete.